demoScripts/aksQuickStart.md

```diff
@@ -0,0 +1,426 @@
+      replicas: 1
+      selector:
+        matchLabels:
+          app: azure-vote-back
+      template:
+        metadata:
+          labels:
+            app: azure-vote-back
+        spec:
+          nodeSelector:
+            "kubernetes.io/os": linux
+          containers:
+          - name: azure-vote-back
+            image: mcr.microsoft.com/oss/bitnami/redis:6.0.8
+            env:
+            - name: ALLOW_EMPTY_PASSWORD
+              value: "yes"
+            resources:
+              requests:
+                cpu: 100m
+                memory: 128Mi
+              limits:
+                cpu: 250m
+                memory: 256Mi
+            ports:
+            - containerPort: 6379
+              name: redis
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: azure-vote-back
+    spec:
+      ports:
+      - port: 6379
+      selector:
+        app: azure-vote-back
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: azure-vote-front
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app: azure-vote-front
+      template:
+        metadata:
+          labels:
+            app: azure-vote-front
+        spec:
+          nodeSelector:
+            "kubernetes.io/os": linux
+          containers:
+          - name: azure-vote-front
+            image: mcr.microsoft.com/azuredocs/azure-vote-front:v1
+            resources:
+              requests:
+                cpu: 100m
+                memory: 128Mi
+              limits:
+                cpu: 250m
+                memory: 256Mi
+            ports:
+            - containerPort: 80
+            env:
+            - name: REDIS
+              value: "azure-vote-back"
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: azure-vote-front
+    spec:
+      type: LoadBalancer
+      ports:
+      - port: 80
+      selector:
+        app: azure-vote-front
+
+    EOF
+    ```
+      For a breakdown of YAML manifest files, see [Deployments and YAML manifests](../concepts-clusters-workloads.md#deployments-and-yaml-manifests).
+
+1. Deploy the application using the [kubectl apply][kubectl-apply] command and specify the name of your YAML manifest:
+
+    ```azurecli-interactive
+    kubectl apply -f azure-vote.yaml
+    ```
+
+    The following example resembles output showing the successfully created deployments and services:
+    <!--expected_similarity=0.8-->
+    ```output
+    deployment "azure-vote-back" created
+    service "azure-vote-back" created
+    deployment "azure-vote-front" created
+    service "azure-vote-front" created
+    ```
+
+## Test the application
+
+When the application runs, a Kubernetes service exposes the application front-end to the internet. This process can take a few minutes to complete.
+
+Monitor progress using the [kubectl get service][kubectl-get] command.
+
+```azurecli-interactive
+timeout 30 kubectl get service azure-vote-front --watch
```

Comment:
The timeout here is for automated testing, right?

I wonder if we should implement a "special command" parser in Innovation Engine. the original SimDem code could do this, it would look for commands that needed to be handled differently in different environments and rewrite them in test. Here we could look for the `--watch` and automatically add the `timeout 30` if not already present.

Doesn't need to be done in this doc but if it is a viable idea we should create an issue for this.

Reply:
I like that idea. However, doesn't that break one of our rules? This was from chat the other day - 

"I'm uncomfortable with the proposal to ignore it. It breaks the rule that we should not have a different behavior in the engine vs standalone execution. This is a very important rule and should always be enforced. Failure to do so means that we are no longer testing the thing that is going into the customers hands."

Are we okay with altering behavior during testing?